### DISCUSSION TITLE
My first map: Friedensau, Zeltplatz

### DISCUSSION THREAD

**Discussion by surffan** [OpenOrienteering/mapper#2068] [Show and tell]
Hello together

I would like to show you here the work on my first OOM project. I would like to achieve the following.

- Get feedback from you guys on what I could improve.
- For you professionals an insight with which problems a beginner struggles.
- Hopefully motivate all beginners to start a project themselves.

So let's start with the basics.

## Project
Map of a campground for scouts
**Location:** Friedensau in Germany (Saxony-Anhalt)
[Location on Google Maps](https://goo.gl/maps/qa8voYuYxQTNc3KX8)
**Idea:** At the end of May there will be a tent camp and I will offer a workshop on orienteering with my group. For this I need of course an appropriate map.

### Selection of scale and symbols
I print the map on A4 paper. So that also a legend finds place I have decided for the scale 1:2,000.
Regarding symbols, I have oriented myself to the [SCOOL project from Switzerland](https://www.scool.ch/de/).
I found a corresponding symbol set on the OCAD website for download.
https://www.ocad.com/en/documents/

### Preliminary work
First of all I got the following basic data:

- Openstreetmap Export
- Ortophoto
- National map 1:25,000

https://www.lvermgeo.sachsen-anhalt.de/de/kostenfreie_geobasisdaten_lvermgeo.html

### First drawing on PC
Based on the maps / photos and my memory I drew a first rough draft on the PC. This looks like this.

![Friedensau_Zeltplatz_01_Basis](https://user-images.githubusercontent.com/707767/167566758-24fa21a1-14e3-4c65-a701-55e9beb0d399.png)

### Work on site with the tablet
Next, I installed the Android version of OOM on my tablet and transferred the base map.
Then, in about 3 hours, I captured details and took photos directly on the tablet on site.

![Friedensau_Zeltplatz_02_Fieldwork](https://user-images.githubusercontent.com/707767/167566846-aa919e88-050b-4313-85cb-9f51c412ddaa.jpg)

Working on the tablet has the great advantage that I can capture all adjustments directly digitally. The GPS position helps me with positioning. However, really clean work does not go here yet.

![Friedensau_Zeltplatz_04_MappingIssues_Tablet](https://user-images.githubusercontent.com/707767/167569205-adb1b7ff-1970-4469-b2a4-26aac952bda6.png)

As an example look at the following points.

1. The sanitary building. Here there are canopies under which you can walk through. Drawing this cleanly on the tablet doesn't work.

2. I have not managed to split paths on the tablet to simply draw a gate / gap in a fence.

3. Distributing points on the basis of a path did not work on the tablet either.

4. The clean alignment of buildings along a path was also too fiddly for me on the tablet.

### Next step
I have now transferred the map back to the PC and am working on drawing points cleanly as described above, smoothing paths, filling holes, etc.

**Comment by surffan** [comment]
Now it's been a while since the initial post.
Time to show you what was created at the end of my first card project.

### Campground
After working on site, I was able to add the rest of the finishing touches on the PC.
This was rather hard work, now that I got used to the program.
![Friedensau_Zeltplatz_2000_scool_v1 0 omap](https://user-images.githubusercontent.com/707767/179608555-e303f46e-8d33-4851-8a01-030b8e2c62e9.png)


### Map with surroundings
For the scout camp we organized for Ascension Day, I wanted to create a second map with more surroundings. This should offer for the Scouts basis for a beautiful small OL.

The basic work based on aerial photos, Openstreetmap data and were very similar. But the work outside was much more exhausting and very time consuming.
It is just quickly several kilometers and hours that you are on the road. After I had walked through the village Friedensau, I have also dedicated myself to the forests around the village.
I had to notice, however, that much more than the paths to map and what is so visible from the path, for me in the short time was definitely not in it.
![20220515_071850](https://user-images.githubusercontent.com/707767/179609256-160006fd-89bc-4595-834c-19bd9b3edb4c.jpg)

I have therefore concentrated on the essentials for a simple orienteering along paths.
![Friedensau_10000_v1 0](https://user-images.githubusercontent.com/707767/179608756-af5f5ac5-edbc-4ee7-9ede-3acd9cd9effc.png)

### The camp
We were able to experience a great scout camp with children and youth. In my workshop orientation, 20 - 30 children have occupied themselves with map and compass.
The posts were actually all found well and no team got lost.

So all in all I am satisfied with the result.

**Many thanks to the team that developed OOM!**

For all those who would like to see the result in detail, you can download the maps here as PDF. 
On request I can send you the .omap file if you are interested. Just write me an email.

Map 1:2.000
[Friedensau_Zeltplatz_1_2000_v1.0_druck.pdf](https://github.com/OpenOrienteering/mapper/files/9135465/Friedensau_Zeltplatz_1_2000_v1.0_druck.pdf)
Map 1:10.000
[Friedensau_1_10000_v_1.0_druck.pdf](https://github.com/OpenOrienteering/mapper/files/9135477/Friedensau_1_10000_v_1.0_druck.pdf)

**Reply by BR-73** [reply]
Hi, really a nice map. How did you make the legenda and how did you add the text? I made a legenda in Photoshop and added it to het map as a png file. But the quality is far to low then.

**Reply by surffan** [reply]
To create a legend, I use the following trick.

1. I create a seperate layer for the legend and title to get the text organized.
2. Then I create a new Text Symbol
 ![grafik](https://github.com/user-attachments/assets/c7e47a22-2d0b-4968-9816-49ab5a4c0878)
 * The trick is now to enhance the line spacing, and also activiate the OCAD compatibility settings.
![grafik](https://github.com/user-attachments/assets/2bcbd20c-9211-488d-8f61-afa01a3f8ec7)
 * In the OCAD settings activate "Line below paragraphs", set the line color and distance.
 * Also create a new tabular position
![grafik](https://github.com/user-attachments/assets/06ea208f-be07-4874-a77c-85ae75273e01)
4. Then place a new text with this symbol on your map. Set the alignment of the text to left and top. Now enter the text like this.
```
TAB    Label 1
TAB    Label 2
``` 
![grafik](https://github.com/user-attachments/assets/8d1f596b-9be2-4b02-b64d-6039715b6434)
5. After entering the text draw the symbols with the normal map drawing tool. 

Thats it.
I added a sample .omap file here (inside .zip)
[TitleAndLegendDemo.zip](https://github.com/user-attachments/files/19934802/TitleAndLegendDemo.zip)

**Reply by BR-73** [reply]
Thanks a lot! I managed to make a legend and added a title and some text.

**Reply by BR-73** [reply]
Another question: how do I make a new layer? I made the legend in the same layer because I didn't find out how to make a new layer. But now I want to draw a route in the map and I want that on a separate layer.

**Reply by surffan** [reply]
### Map parts

The "Layers" are called Map Parts in OOM, you can create a new one via the menue and then switch between the different part via dropdown list in the toolbar.

![grafik](https://github.com/user-attachments/assets/ef541065-aa7c-4239-9436-b61463aad18e)

### Drawing routes

For drawing routes, I use the tool [purple pen](https://purple-pen.org/). In purple pen, I can import an open orienteering map or also a pdf of a map and draw several courses. Also here the printing of a set of maps is very easy.

![grafik](https://github.com/user-attachments/assets/6aaebfd1-d26f-4961-bb55-55cf4fe6a79c)

**Reply by dg0yt** [reply]
> The "Layers" are called Map Parts in OOM

This is the right direction. But I would like to note that all parts are drawn simultaneously. There is no "stacking" as one might expect for "layers".

**Comment by rhansson** [comment]
Great work! I have a few questions for you. Expect an email at the @klenk address. 
Roland

**Comment by surffan** [comment]
This spring I updated the 1:10.000 map and spent another day in the woods to improve accuracy in the wooded area.

![image](https://github.com/OpenOrienteering/mapper/assets/707767/2a9b7ee2-c594-4459-9a58-3276c2e9be37)


[Friedensau_1_10000_v_1.2_druck.pdf](https://github.com/OpenOrienteering/mapper/files/12172825/Friedensau_1_10000_v_1.2_druck.pdf)

**Comment by surffan** [comment]
New year, new map version.

I'm still a happy user of the open orienteering software running on my PC and on my Android tablet for the field work.
So I invested again some hours to update my map of Friedensau.

![v1 2_vs_1 3](https://github.com/OpenOrienteering/mapper/assets/707767/49134da7-11e5-425e-9682-3fb460cf56a0)

[Friedensau_1_10000_v_1.3_druck.pdf](https://github.com/OpenOrienteering/mapper/files/14854427/Friedensau_1_10000_v_1.3_druck.pdf)

**Comment by surffan** [comment]
New year, new enhanced map version for 2025.
[Friedensau_1_10000_v_1.4_druck.pdf](https://github.com/user-attachments/files/19906940/Friedensau_1_10000_v_1.4_druck.pdf)

**Comment by rhansson** [comment]
This tool can be of help in adding external vector graphics:
https://github.com/rhansson/svg2omap
--
Roland

**Reply by surffan** [reply]
Thank you for the tipp. I will deffinitly check out the tool to add logos to my maps.
